src/store/modules/auth.js

```diff
@@ -10,10 +13,12 @@ const state = {
   reAuthenticationState: "not-started",
   reAuthenticationPromise: null,
   reAuthenticationPromiseResolver: null,
+  userSettings: null,
+  workspaceSettings: null,
 };
 
 const getters = {
-  isAuthenticated: (state) => !!state.accessToken,
+  isAuthenticated: (state) => !!state.userId,
```

Comment:
Haan so isAuthenticated was being set pre-maturely.
If the access token was present in the store but the process of fetching the user was still underway, even then it marked `isAuthenticated` as true.

And I needed this because I wanted to know when a user has been fetched, so I can pick up it's config.
Before this change, when i was watching `isAuthenticated`, it would trigger before the user was being set.